### PR TITLE
Date picker

### PR DESCRIPTION
**Pull request by henrixapp**

### CODE REVIEW

File: app/views/assignments/_form.html.erb [hunk 1]
```diff
@@ -11,9 +11,14 @@
...
       <div class="form-group col-3">
-        <%= f.datetime_field :deadline,
+        <%= f.text_field :deadline,
                              class: 'form-control',
                              id: "assignment_deadline_#{assignment.id}" %>
+        <script type="text/javascript">
+         $("#assignment_deadline_<%= @assignment.id %>").datetimepicker({ format:'d.m.Y H:i',
+  inline:false,
+});
+        </script>
```
Comment (fosterfarrell9) [inline app/views/assignments/_form.html.erb:21]: Is it possible to do this unobtrusive like in some extra pack via webpacker  (or in app/asstes/javascript via sprockets)?

**Review by fosterfarrell9 — CHANGES_REQUESTED** [review]
Works like a charm.

File: app/views/assignments/_form.html.erb (hunk 1 above)
Comment (henrixapp) [reply app/views/assignments/_form.html.erb:21]: Yes, it must be wrapped in a document ready function. I placed it there, that is obvious what is happening.

Comment (fosterfarrell9) [reply app/views/assignments/_form.html.erb:21]: I would rather like it in a separate datetimepicker.js or. coffee, and then wrapped inside a turbolinks:load funktion.

Comment (henrixapp) [reply app/views/assignments/_form.html.erb]: The problem is, that the datepickers are not available in the dom in the suggested event. I have moved it to a seperate file, but this makes it not obvious, when it is needed.

**Comment by henrixapp** [comment]
Closed in favour of #190

**Pull request closed without merging**